src/app/components/Video/index.jsx

```diff
@@ -50,15 +41,49 @@ const Video = ({
       />
     </Helmet>
     <MediaPlayerContainer id={id} />
-    <div>title: {title}</div>
-    <div>statsAppName: {statsAppName}</div>
-    <div>statsAppType: {statsAppType}</div>
-    <div>statsCountername: {statsCountername}</div>
-    <div>statsDestination: {statsDestination}</div>
-    <div>uiLocale: {uiLocale}</div>
   </>
 );
 
-Video.propTypes = videoComponentPropTypes;
+Video.propTypes = {
```

Comment:
Moved proptypes into the component for ease of reference and since we'll be moving this component out of Simorgh and into Psammead shortly.